articles/testing-domain.mdx

```diff
@@ -0,0 +1,49 @@
+---
+_schema: default
+_uuid: a68dc6a3-6e01-4af9-b8b1-75d4ee6a705b
+_created_at: '2021-09-15 23:13:40 +0000'
+title: Testing Domain
+nav_title: Testing Domain
+published: true
+image: >-
+  https://cc-dam.imgix.net/documentation/images/default-thumbnail/2023-Q4/CloudCannon-Documentation-Thumbnail.png
+description: Learn about the free .cloudvent.net Testing Domain.
+article_category:
+  - Explanation
+article_topic:
+  - Sites
+tags: []
+related_articles:
+related_links: []
+---
+Every Site on CloudCannon is assigned a free Testing Domain with the *.cloudvent.net* URL. Each Testing Domain uses a randomly generated adjective and noun separated by a hyphen (i.e., [grey-grouse.cloudvent.net](https://grey-grouse.cloudvent.net/) or [brave-submarine.cloudvent.net](https://brave-submarine.cloudvent.net/)).
+
+A Testing Domain is useful for seeing your content live after you have a successful [Site build](/documentation/articles/what-is-a-site-build/). If you use a [publishing workflow](/documentation/guides/staging-workflow-guide/), your Testing Domain will allow you to see the content for each branch on it's own domain.
+
+You can open your Testing Domain any time after a successful build by clicking on the *Live Site preview* button in your [*Site Navigation*](/documentation/articles/configure-your-site-navigation/). The button will take you to the home page of your Site on your Testing Domain.
+
+<comp.DocsImage path="https://cc-dam.imgix.net/documentation/images/editing/navigation/2024-10/CloudCannon-Documentation-Site-Navigation-Header.png" alt="A screenshot of the Site Navigation Header shows the Live Site preview button, which allows you to open your Testing Domain." title="The Site Navigation" type="ui-snippet"/>
+
+You can also open specific pages on your Testing Domain from any output file by clicking on the *Preview live Site here* option in the *Context menu* in an editing interface or *Collection browser*.
+
+<comp.DocsImage path="https://cc-dam.imgix.net/documentation/images/editing/collections/2023-Q4/CloudCannon-Documentation-File-Context-Menu.png" alt="A screenshot of the Articles Collection browser shows the Context Menu for a file with the option to Preview live Site here." title="The Context Menu" type="ui-snippet"/>
+
+<comp.Notice info_type="important">
+  To stop serving content on your Testing Domain, you can either [set a password](/documentation/articles/adding-password-authentication-to-your-site/) to prevent or contact our friendly [support team](/support/).
```

Comment:
`to prevent` here seems out of place.
Another option to remove the Testing Domain is by using `Headless mode`